project/Dependencies.scala

```diff
@@ -19,12 +19,13 @@ object Dependencies {
   val scalaTest =  "org.scalatestplus" %% "play" % "1.4.0-M4" % "test"
   val scalaz = "org.scalaz" %% "scalaz-core" % "7.1.1"
   val selenium = "org.seleniumhq.selenium" % "selenium-java" % "2.48.2" % "test"
+  val specs2Extra = "org.specs2" % "specs2-matcher-extra_2.11" % "3.6" % "test"
```

Comment:
Can you use the [`%%`](http://www.scala-sbt.org/0.13/docs/Library-Dependencies.html#Getting+the+right+Scala+version+with) syntax here, ie: `"org.specs2" %% "specs2-matcher-extra" % "3.6" % "test"`